charts/fluentd-elasticsearch/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: fluentd-elasticsearch
-version: 2.9.1
+version: 2.9.2
```

Comment:
new feature should raise minor version. -> 2.10.0

Reply:
Ok I put 2.10.0